src/proto/plumtree.rs

```diff
@@ -427,8 +444,16 @@ impl<PI: PeerIdentity> State<PI> {
 
     /// Dispatches messages from lazy queue over to lazy peers.
     fn on_dispatch_timer(&mut self, io: &mut impl IO<PI>) {
+        let chunk_size = self.max_message_size
+            // Space for discriminator
+            - 1
+            // Space for length prefix
+            - 2;
+        let chunk_len = chunk_size / IHave::POSTCARD_MAX_SIZE;
```

Comment:
we need to make sure this is non zero because `list.chunks` two lines below panics in that case

Reply:
Now that `max_message_size` is guaranteed to be above 512 it can never be zero I think.